brainiak/hyperparamopt/hpo.py

```diff
@@ -0,0 +1,231 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Hyper Parameter Optimization (HPO)
+
+This implementation is based on the work:
+
+.. [Bergstra2011] "Algorithms for Hyper-Parameter Optimization",
+   James S. Bergstra and Bardenet, R\'{e}mi and Bengio, Yoshua
+   and Bal\'{a}zs K\'{e}gl. NIPS 2011
+
+.. [Bergstra2013] "Making a Science of Model Search:
+   Hyperparameter Optimization in Hundreds of Dimensions for
+   Vision Architectures", James Bergstra, Daniel Yamins, David Cox.
+   JMLR W&CP 28 (1) : 115–123, 2013
+
+"""
+
+# Authors: Narayanan Sundaram (Intel Labs)
+
+import logging
+from .mcmc import get_multichain_samples
+from .norm import getgmmpdf
+import numpy as np
+from scipy.special import erf
+from tqdm import tqdm
+
+
+logger = logging.getLogger(__name__)
+
+
+def getsigma(x, minlimit=-np.inf, maxlimit=np.inf):
```

Comment:
Should be get_sigma. Generally, please follow https://www.python.org/dev/peps/pep-0008/#naming-conventions, there are several other naming issues in this PR.